src/test/ui/generic-associated-types/parse/trait-path-type-error-once-implemented.stderr

```diff
@@ -1,8 +1,24 @@
-error: generic associated types in trait paths are currently not implemented
-  --> $DIR/trait-path-type-error-once-implemented.rs:8:30
+warning: the feature `generic_associated_types` is incomplete and may not be safe to use and/or cause compiler crashes
+  --> $DIR/trait-path-type-error-once-implemented.rs:1:12
+   |
+LL | #![feature(generic_associated_types)]
+   |            ^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = note: `#[warn(incomplete_features)]` on by default
+   = note: see issue #44265 <https://github.com/rust-lang/rust/issues/44265> for more information
+
+error[E0107]: wrong number of lifetime arguments: expected 1, found 0
+  --> $DIR/trait-path-type-error-once-implemented.rs:9:29
+   |
+LL |   fn f2<'a>(arg : Box<dyn X<Y<1> = &'a ()>>) {}
+   |                             ^^^^^^^^^^^^^ expected 1 lifetime argument
```

Comment:
I'm noticing: the span here seems wrong? Should it be on the `1`?

Reply:
Yes, but this wasn't related to this PR. This was an error in `check_generic_arg_count`, which was modified since this PR, not sure whether the wrong span is still used there.